psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -74,25 +74,19 @@
 #include <omp.h>
```

Comment:
Is `c-f`ing this a good idea at the moment due to the large changes caused by #847? Same with the `libmints` folder, there are several PR's making deep changes there.

Reply:
No, it's not. I'll de-format before un-WIP-ping.